projects/hslayers/src/components/add-data/catalogue/add-data-list-item.component.ts

```diff
@@ -83,8 +85,10 @@ export class HsAddDataListItemComponent {
 
   /**
    * @description For a stringified type of service, it returns its description
+   * @param module
```

Comment:
pls don't do this. It's obviously useless. Except for mutating ESLint errors, of course..

Reply:
Autogenerated I guess

Reply:
Yes exactly. I wouldn't bother adding that manually :)

Reply:
Curious. What tool are you using to autofix lint errors? Is it some VS Code setting/extension doing?

Reply:
Actually that was jsut eslint via 'Fix all auto-fixable problems'

Reply:
![image](https://user-images.githubusercontent.com/44566616/114344713-9c841680-9b60-11eb-9fd1-441c76d6b16c.png)

Reply:
Oh sure, I see. I never use this option, just because of that. Instead I usually choose to 'Fix all prettier/prettier problems'. Ok, thanks for explaining. After you rebase and reinstall, you should not see these JSDoc errors any longer.